functional/basic-attestation-on-localhost/test.sh

```diff
@@ -89,6 +91,7 @@ _EOF"
             cat ${HTTP_SERVER_LOG}
             rlAssertGrep '\\"type\\": \\"revocation\\", \\"ip\\": \\"127.0.0.1\\", \\"agent_id\\": \\"d432fbb3-d2f1-4a97-9ef7-75bd81c00000\\"' ${HTTP_SERVER_LOG} -i
             rlAssertNotGrep ERROR ${HTTP_SERVER_LOG} -i
+            rlAssertGrep 'root' /var/lib/keylime/sudo_script_output.txt -i
```

Comment:
I think you are not removing that file in CleanUp phase.

Reply:
Now, file is removed in clean up phase.